cmd/root.go

```diff
@@ -58,6 +58,8 @@ func AddCommands(cmd *cobra.Command) {
 		config.BlockCacheSize, "size in layers of meshdb block cache")
 	cmd.PersistentFlags().StringVar(&config.PublishEventsURL, "events-url",
 		config.PublishEventsURL, "publish events to this url; if no url specified no events will be published")
+	cmd.PersistentFlags().BoolVar(&config.Profiler, "profiler",
+		config.Profiler, "enable GCP profiler")
```

Comment:
pprof isn't actually a GCP specific profiler, although GCP supports it. It's a general purpose Go profiler that can be used locally, as well. I'd remove the reference to GCP from the comment here.

Reply:
Done